meson.build

```diff
@@ -0,0 +1,59 @@
+project(
+    'powervm-handler',
+    'cpp',
+    default_options: [
+        'buildtype=debugoptimized',
+        'cpp_std=c++20',
+        'warning_level=3',
+        'werror=true',
+    ],
+    meson_version: '>=0.57.0',
+)
+
+cpp = meson.get_compiler('cpp')
+
+sdbusplus_dep = dependency(
+    'sdbusplus',
+    fallback: ['sdbusplus', 'sdbusplus_dep'],
+)
+
+sdeventplus_dep = dependency(
+    'sdeventplus',
+    fallback: ['sdeventplus', 'sdeventplus_dep'],
+)
+
+phosphor_dbus_interfaces_dep = dependency(
+    'phosphor-dbus-interfaces',
+    fallback: [
+        'phosphor-dbus-interfaces',
+        'phosphor_dbus_interfaces_dep'
+    ],
+)
+
+phosphor_logging_dep = dependency(
+    'phosphor-logging',
+    fallback: ['phosphor-logging', 'phosphor_logging_dep'],
+)
+
+systemd_dep = dependency('systemd')
+
+configure_file(
+    input: 'config.h.in',
+    output: 'config.h',
+    configuration: configuration_data(),
+)
+
+dump_offload_deps = [
```

Comment:
Just curiosity, Looks you used the `fmt` library but, did not add the dependency? is it added as part of the C++ std library?

Reply:
yeah good observation, not sure where it is getting from, it did not complain during build.